siliconcompiler/report/utils.py

```diff
@@ -28,7 +28,8 @@ def _collect_data(chip, flow, steplist):
     # Gather data and determine which metrics to show
     # We show a metric if:
     # - it is not in ['option', 'metricoff'] -AND-
-    # - at least one step in the steplist has a non-zero weight for the metric -OR -
+    # - at least one step in the steplist has a non-zero weight for the metric
+    #  -OR -
```

Comment:
Small thing, but I think the line length changes in this file (here and the two below) shouldn't be necessary, these lines were linting clean previously. This might have happened if you used a formatting tool that has a slightly stricter line length requirement than we lint for. It would be nice to revert these couple changes, since I think wrapping lines more than necessary can impact readability a bit.